install/Knative-with-OpenShift.md

```diff
@@ -11,168 +11,154 @@ You can find [guides for other platforms here](README.md).
 
-## Creating a new OpenShift cluster
+## Configuring `oc` (openshift cli)
 
-Create a new OpenShift cluster on your local machine using `oc cluster up`:
+Running the following command make sure that we have right version of `oc` and configured the DOCKER daemon to be connected to minishift Docker.
 
 ```shell
-oc cluster up --write-config
-
-# Enable admission webhooks
-sed -i -e 's/"admissionConfig":{"pluginConfig":null}/"admissionConfig": {\
-    "pluginConfig": {\
-        "ValidatingAdmissionWebhook": {\
-            "configuration": {\
-                "apiVersion": "v1",\
-                "kind": "DefaultAdmissionConfig",\
-                "disable": false\
-            }\
-        },\
-        "MutatingAdmissionWebhook": {\
-            "configuration": {\
-                "apiVersion": "v1",\
-                "kind": "DefaultAdmissionConfig",\
-                "disable": false\
-            }\
-        }\
-    }\
-}/' openshift.local.clusterup/kube-apiserver/master-config.yaml
-
-oc cluster up --server-loglevel=5
+eval $(minishift docker-env) && eval $(minishift oc-env)
 ```
 
-Once the cluster is up, login as the cluster administrator:
+## Preparing Knative Deployment
+
+### Enable Admission Controller Webhook
+To be able to deploy and run serverless Knative applications, its required that we enable the [Admission Controller Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).  
+
+Run the following command to make OpenShift (run via minishift) to be configured for [Admission Controller Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/):
 
 ```shell
-oc login -u system:admin
+minishift openshift config set --patch '{
+    "admissionConfig": {
+        "pluginConfig": {
+            "ValidatingAdmissionWebhook": {
+                "configuration": {
+                    "apiVersion": "v1",
+                    "kind": "DefaultAdmissionConfig",
+                    "disable": false
+                }
+            },
+            "MutatingAdmissionWebhook": {
+                "configuration": {
+                    "apiVersion": "v1",
+                    "kind": "DefaultAdmissionConfig",
+                    "disable": false
+                }
+            }
+        }
+    }
+}'
 ```
 
-Now, we'll set up the default project for use with Knative.
+Please allow few minutes for the OpenShift to be restarted.
 
-```shell
-oc project default
+### Configuring a OpenShift project
 
-# SCCs (Security Context Constraints) are the precursor to the PSP (Pod
-# Security Policy) mechanism in Kubernetes.
-oc adm policy add-scc-to-user privileged -z default -n default
+1. Since there was a `admin` is added via `admin-user` addon during minishift configuration, its now possible to login with the user like:
```

Comment:
"Since there was a `admin` is added via `admin-user` addon during minishift configuration, its now possible to login with the user like:"
-->
"Since there was an `admin-user` addon added during minishift configuration, it's now possible to login with the user. For example:"

Reply:
done!